FishBunDemo/src/main/java/com/sangcomz/fishbundemo/WithActivityActivity.kt

```diff
@@ -0,0 +1,160 @@
+package com.sangcomz.fishbundemo
+
+import android.content.Intent
+import android.graphics.Color
+import android.net.Uri
+import android.os.Bundle
+import android.view.Menu
+import android.view.MenuItem
+import android.widget.ImageView
+import androidx.appcompat.app.AppCompatActivity
+import androidx.core.content.ContextCompat
+import androidx.recyclerview.widget.LinearLayoutManager
+import androidx.recyclerview.widget.RecyclerView
+import com.sangcomz.fishbun.FishBun
+import com.sangcomz.fishbun.adapter.image.impl.GlideAdapter
+import com.sangcomz.fishbun.adapter.image.impl.PicassoAdapter
+import com.sangcomz.fishbun.define.Define
+import java.util.*
+
+class WithActivityActivity : AppCompatActivity() {
+
+    var path = ArrayList<Uri>()
+    private lateinit var imgMain: ImageView
+    private lateinit var recyclerView: RecyclerView
+    private lateinit var linearLayoutManager: LinearLayoutManager
+    private lateinit var imageAdapter: ImageAdapter
+    private lateinit var mainController: ImageController
+    var mode: Int = 0
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_withactivity)
+
+        mode = intent.getIntExtra("mode", -1)
+        imgMain = findViewById(R.id.img_main)
+        recyclerView = findViewById(R.id.recyclerview)
+
+        linearLayoutManager = LinearLayoutManager(this, LinearLayoutManager.HORIZONTAL, false)
+        mainController = ImageController(imgMain)
+        imageAdapter = ImageAdapter(mainController, path)
+        recyclerView.layoutManager = linearLayoutManager
+        recyclerView.adapter = imageAdapter
+    }
+
+    override fun onActivityResult(
+        requestCode: Int, resultCode: Int,
+        imageData: Intent?
+    ) {
+        super.onActivityResult(requestCode, resultCode, imageData)
+
+        when (requestCode) {
+            Define.ALBUM_REQUEST_CODE -> if (resultCode == RESULT_OK) {
+                path = imageData!!.getParcelableArrayListExtra(Define.INTENT_PATH)
```

Comment:
I can't understand what you exactly mean. Could you show me a suggestion code like above?

Reply:
Sorry, I talked insufficient.
If requestCode are in much ways, Check duplicate `Activity.RESULT_OK` logic.
But It would be change by business logic. So It's OK now yet.

```kotlin
 override fun onActivityResult(requestCode: Int, resultCode: Int, data: Intent?) {
        when() {
              A -> if (resultCode == Activity.RESULT_OK) ...
              B -> if (resultCode == Activity.RESULT_OK) ... else ...
              C -> if (resultCode == Activity.RESULT_OK) ... else ...
       }
}
```

Reply:
Now I understand what you are saying. Thanks.